doc/snippets/Microsoft.Data.SqlClient/SqlBatch.xml

```diff
@@ -163,9 +163,6 @@
                       batch.BatchCommands.Add(batchCommand);
                   }
           
-                  // Optionally Prepare
-                  batch.Prepare();
```

Comment:
I decided to remove this because [SqlBatch.Prepare](https://github.com/dotnet/SqlClient/blob/main/src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlBatch.cs#L116) is a no-op.